<ui version="4.0" >
 <class>PocketDialog</class>
 <widget class="QDialog" name="PocketDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>319</width>
    <height>146</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Create binding pocket</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="label" >
       <property name="text" >
        <string>Method</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="methodCombo" >
       <property name="minimumSize" >
        <size>
         <width>225</width>
         <height>0</height>
        </size>
       </property>
       <property name="toolTip" >
        <string>Method</string>
       </property>
       <item>
        <property name="text" >
         <string>create pocket from current selection</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>create pocket from residue name</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="label_2" >
       <property name="text" >
        <string>Residue name</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="resnameText" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize" >
        <size>
         <width>225</width>
         <height>0</height>
        </size>
       </property>
       <property name="toolTip" >
        <string>The residue name</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="label_3" >
       <property name="text" >
        <string>Radius</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="radiusSpin" >
       <property name="minimumSize" >
        <size>
         <width>225</width>
         <height>0</height>
        </size>
       </property>
       <property name="toolTip" >
        <string>All atoms within radius from the ligand will be part of the binding pocket</string>
       </property>
       <property name="minimum" >
        <double>3.000000000000000</double>
       </property>
       <property name="value" >
        <double>10.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::NoButton|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>PocketDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PocketDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
